src/ontology/cl-edit.owl

```diff
@@ -9536,14 +9536,17 @@ SubClassOf(obo:CL_0000705 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_000097
 
 # Class: obo:CL_0000706 (choroid plexus epithelial cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "Specialized ependymal cell that produces the cerebrospinal fluid from the blood and secretes it into the lumen of the brain and spinal chord.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "GOC:add") Annotation(oboInOwl:hasDbXref "GOC:tfm") Annotation(oboInOwl:hasDbXref "JB:jb") Annotation(oboInOwl:hasDbXref "PMID:15561411") Annotation(oboInOwl:hasDbXref "PMID:9550134") obo:IAO_0000115 obo:CL_0000706 "A specialized ependymal cell that is part of the choroid plexus epithelium, responsible for producing cerebrospinal fluid (CSF) from the blood and secreting it into the brain and spinal cord. The cell is characterized by a brush border on its apical surface, which enhances the secretion of CSF.")
 AnnotationAssertion(terms:date obo:CL_0000706 "2023-08-04T08:48:30Z"^^xsd:dateTime)
 AnnotationAssertion(Annotation(oboInOwl:hasDbXref "DOI:10.1101/2022.10.12.511898") oboInOwl:hasBroadSynonym obo:CL_0000706 "choroid plexus cell")
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_0000706 "FMA:70549")
 AnnotationAssertion(oboInOwl:hasExactSynonym obo:CL_0000706 "epithelial cell of choroid plexus")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:25729351") rdfs:comment obo:CL_0000706 "choroid plexus epithelial cell possesses non-motile 9+0 cilia. While these cilia can exhibit transient motility during development, they generally do not contribute significantly to the directional flow of cerebrospinal fluid (CSF). Instead, they are primarily involved in sensory functions.")
```

Comment:
If you want to avoid capitalizing the cell label, you can add an "A": A choroid plexus epithelial cell possesses non-motile 9+0 cilia.